docs/src/main/sphinx/release/release-400.md

```diff
@@ -9,6 +9,21 @@
 
 * Add support for [truncating tables](/sql/truncate). ({issue}`14494`)
 
+## Delta Lake connector
+
+* Prevent coordinator out-of-memory failure when querying a large number of
+  tables in a short period of time. ({issue}`14571`)
+
+## Hive connector
+
+* Reduce memory usage when scanning a large number of partitions, and add the
+  `hive.max-partitions-for-eager-load` configuration property to manage the
+  number of partitions which can be loaded into memory. ({issue}`14225`)
+* Increase the default value of the `hive. max-partitions-per-scan`
+  configuration property to 1,000,000 from 100,000. ({issue}`14225`)
+* Utilize the `hive.metastore.thrift.delete-files-on-drop` configuration
```

Comment:
What does this mean? Did it exist but it was not being used? That would be a bug.

Reply:
It existed but was only being used for table drops, not partition drops.